onboarding/src/Components/Dashboard/Employee.js

```diff
@@ -1,30 +1,24 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import LoggedUser from "../hooks/LoggedUser.js";
 
-
-const Employee = ({ employee, loggedUser }) => {
-  let avatar = "/onboarding/static/images/unknown-profile.jpg", loggedUserCp;
+const Employee = ({ employee }) => {
+  let avatar = "/onboarding/static/images/unknown-profile.jpg";
   if(employee.avatar && employee.avatar.length > 1)
     avatar = employee.avatar;
```

Comment:
Extra verification because avatar URL is stored in `sessionStorage`.
https://github.com/binawork/OnlineOnboarding/pull/174/files#r569395358